style.css

```diff
@@ -0,0 +1 @@
+h1 { color: brown; }
```

Comment:
- [ ] Kindly use the class name given to your h1 tag to style the content

E.g
```CSS
.title {
  color: brown;
}
```